nautobot/dcim/forms.py

```diff
@@ -5323,11 +5329,15 @@ def __init__(self, *args, **kwargs):
             self.fields["device"].disabled = True
             self.fields["device"].required = False
 
+        self.initial["vrfs"] = self.instance.vrfs.values_list("id", flat=True)
```

Comment:
Do we need something similar for `initial["interfaces"]`? If not, why not?

Reply:
Seems like the initial["interfaces"] is populated correctly. Is it because `interfaces` on VDC is a forward m2m relationship?

Reply:
Could be, yeah - forward vs reverse relationship.